### DISCUSSION TITLE
Archive a single page from published space

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#416] [Feature Requests]
I would like to be able archive or unpublish a single page from content that has been already merged into a published space. 

At the moment this isn't possible within GitBook unless I use a workaround by manually copying and pasting the content into a draft CR or copying and pasting it to a different (unpublished) space. 
I would like this feature to be available within GitBook.

**Comment by MadeByPP** [comment]
What permissions are given to you?
When you press the edit button, a change request is created and the document can be archived. It seems you can't just archive a single page.
Or did I misunderstand the feature request?

**Reply by taszap** [reply]
This request was created by GitBook staff for a user. 
You refer to a content that isn't merged but what this user wanted was an option to archive a single page from **already merged** content. 

They could possibly use a revert function but:
- the page or pages that the customer wants to "hide" would have to have been created in a single CR
- they would risk overriding other changes that were merged in subsequent CRs 

I will clarify this!

**Reply by MadeByPP** [reply]
sorry. I didn't see the Maintainer sign next to the username.

**Comment by aikoii** [comment]
Related #12 and #22

**Reply by taszap** [reply]
I will close this in favour of #22
